packages/react-server/core/util/ClientCssHelper.js

```diff
@@ -96,14 +96,18 @@ module.exports = {
 }
 
 function normalizeLocalUrl(url) {
-	const urlBase = location.protocol + "//" + location.host;
+	const urlBase = location.host + (location.port ? ':' + location.port : '');
```

Comment:
I think `host` should actually have the port included.

https://developer.mozilla.org/en-US/docs/Web/API/Location
![image](https://cloud.githubusercontent.com/assets/115908/24931900/f3640956-1ec3-11e7-8510-073202e94430.png)